app/processing/transform-payment-request.js

```diff
@@ -7,11 +7,12 @@ const transformPaymentRequest = async (paymentRequest) => {
   // if yes, need to treat as post payment adjustment and calculate Delta which can result in payment request splitting
   const previousPaymentRequests = await getCompletedPaymentRequests(paymentRequest.schemeId, paymentRequest.frn, paymentRequest.marketingYear, paymentRequest.agreementNumber, paymentRequest.paymentRequestNumber)
   if (previousPaymentRequests.length) {
-    const updatedPaymentRequests = calculateDelta(paymentRequest, previousPaymentRequests)
-    return enrichPaymentRequests(updatedPaymentRequests, previousPaymentRequests)
+    const deltaPaymentRequests = calculateDelta(paymentRequest, previousPaymentRequests)
```

Comment:
Could pull this out as `const { deltaPaymentRequests, completedPaymentRequests } = calculateDelta(paymentRequest, previousPaymentRequests)` to match what `calculateDelta` is producing then pulling it together again for the return, `return { deltaPaymentRequests, completedPaymentRequests}`